dex-it/src/test/scala/com/wavesplatform/it/sync/ReBroadcastingFailedByScriptTxsTestSuite.scala

```diff
@@ -0,0 +1,98 @@
+package com.wavesplatform.it.sync
+
+import com.typesafe.config.{Config, ConfigFactory}
+import com.wavesplatform.dex.api.http.entities.HttpOrderStatus
+import com.wavesplatform.dex.domain.account.KeyPair
+import com.wavesplatform.dex.domain.asset.Asset.Waves
+import com.wavesplatform.dex.domain.bytes.ByteStr
+import com.wavesplatform.dex.domain.order.OrderType
+import com.wavesplatform.dex.it.test.Scripts
+import com.wavesplatform.it.MatcherSuiteBase
+
+import scala.concurrent.duration._
+
+final class ReBroadcastingFailedByScriptTxsTestSuite extends MatcherSuiteBase {
+
+  "ReBroadcastingFailedByScriptTxsTestSuite" - {
+
+    "should not rebroadcast transactions failed by account script" in {
+      val carolOrder = mkOrder(carol, wavesUsdPair, OrderType.SELL, 10.waves, 10.usd, version = 2)
+      dex1.api.place(carolOrder)
```

Comment:
Let's place and await for placement: `placeAndAwaitAtDex(carolOrder)`

Reply:
fxd